bin/check-ping.rb

```diff
@@ -75,9 +81,10 @@ class CheckPING < Sensu::Plugin::Check::CLI
   def run
     result = []
     pt = Net::Ping::External.new(config[:host], nil, config[:timeout])
+    config[:ipv6] ? (ping = pt.ping6) : (ping = pt.ping)
```

Comment:
I think this is easy to implement for IPv4 and IPv6 addresses, but there is no way to auto detect FQDNs reliably.

In consequence there would be a use case when it is unclear if the ping check is going to use IPv4 or IPv6:

check_ping -h 127.0.0.1 -> IPv4
check_ping -h [::1] -> IPv6
check_ping -h localhost -> ???

I believe it is not valid to have an undefined or random behaviour, since it is a different test case to monitor IPv4 or IPv6 connectivity. It is also dangerous to build this using some black magic like using IPv6 and fall back to IPv4, since this is not what we want to monitor: Check if there is IPv6-connectivity.

I think it is a valid decision not to auto detect the use of IPv4 or IPv6 (and detect it wrong in the worse case scenario, resulting in a incorrect test case), but to explicitly request for IPv6 checks. Furthermore, all platform tools like `ping` or `ip` make use of explicit shell options (ping6 is even a complete different implementation) to avoid unclear situations:

```bash
stefan@provornyj ~/sensu-plugins-network-checks $ ping [::1]
ping: unknown host [::1]
```